src/scss/components/beacon/index.scss

```diff
@@ -0,0 +1,46 @@
+/// ========== VARIABLES ========== ///
+@import "vars";
+
+/// ========== hxBeacon ========== ///
+hx-beacon-circle {
```

Comment:
Please try moving this to the Shadow DOM styles - `src/elements/hx-beacon/_shadow.scss`.

Also,  rename to `#hxBeaconPulse`.

Reply:
Changes have been done, please review